etl/steps/data/meadow/wb/2023-07-10/education.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from owid.catalog import Table
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Create a PathFinder instance for the current file
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    """
+    Main function to load, process and save all World Bank Education datasets.
+
+    """
+    snap: Snapshot = paths.load_dependency("education.csv")
+
+    # Load data from snapshot
+    df = pd.read_csv(snap.path)
+    df.rename(columns={"Series": "indicator_name", "wb_seriescode": "indicator_code"}, inplace=True)
+
+    tb = Table(df, short_name=paths.short_name, underscore=True)
+    tb.set_index(["country", "year", "indicator_name"], inplace=True)
```

Comment:
Consider using `verify_integrity=True`. If it fails, maybe explicitly write `verify_integrity=False` and add a comment above, mentioning that non-unique indexes are expected.
Also (for the future) I think it's generally better to avoid using `inplace=True`. But you don't need to change those now.